su2AdapterInstall

```diff
@@ -51,38 +41,15 @@ cp ./adapter_files/precice.cpp $SU2_HOME/SU2_CFD/src || { printf >&2 "Cannot cop
 printf " done.\n\n"
 
-# Add additional files
-printf "Adding additional files..."
-cp ./add_files/ax_cxx_compile_stdcxx.m4 $SU2_HOME/m4 || { printf >&2 "Cannot copy add_files/ax_cxx_compile_stdcxx.m4. Is variable SU2_HOME set correctly? Are you running the script from the correct directory?\nAborting.\n"; exit 1; }
-printf " done.\n\n"
-
-# Replace configure and build related files
-printf "Replacing original configure and build related files..."
-cp ./replacement_files/aclocal.m4 $SU2_HOME || { printf >&2 "Cannot copy replacement_files/aclocal.m4. Is variable SU2_HOME set correctly? Are you running the script from the correct directory?\nAborting.\n"; exit 1; }
-cp ./replacement_files/configure.ac $SU2_HOME || { printf >&2 "Cannot copy replacement_files/configure.ac. Is variable SU2_HOME set correctly? Are you running the script from the correct directory?\nAborting.\n"; exit 1; }
-cp ./replacement_files/Makefile.am $SU2_HOME/SU2_CFD/obj || { printf >&2 "Cannot copy replacement_files/Makefile.am. Is variable SU2_HOME set correctly? Are you running the script from the correct directory?\nAborting.\n"; exit 1; }
-printf " done.\n\n"
-
-# Autoreconfigure SU2
-printf "Autoreconfiguring due to the changed configure and build related files..."
-cd $SU2_HOME
-autoreconf
+# Replace SU2_CFD/src/meson.build
+printf "Replacing original build related file..."
+#cp ./replacement_files/aclocal.m4 $SU2_HOME || { printf >&2 "Cannot copy replacement_files/aclocal.m4. Is variable SU2_HOME set correctly? Are you running the script from the correct directory?\nAborting.\n"; exit 1; }
+#cp ./replacement_files/configure.ac $SU2_HOME || { printf >&2 "Cannot copy replacement_files/configure.ac. Is variable SU2_HOME set correctly? Are you running the script from the correct directory?\nAborting.\n"; exit 1; }
+cp ./replacement_files/meson.build $SU2_HOME/SU2_CFD/src || { printf >&2 "Cannot copy replacement_files/meson.build. Is variable SU2_HOME set correctly? Are you running the script from the correct directory?\nAborting.\n"; exit 1; }
 printf " done.\n\n"
 
 # Output to guide the user
 printf "\nPlease navigate to the SU2 home directory $SU2_HOME to configure and build SU2 as described in its installation description.\n"
 printf "Please enable MPI when configuring SU2.\n\n"
 
-# If user already configured SU2 before, print the command as reminder
-if [ -f $SU2_HOME/config.log ]
-then
-    printf "Last time you configured SU2 by invoking:\n"
-    sed '1,10!d' $SU2_HOME/config.log | grep -F "$ ./configure"
-    printf "You may want to reuse this command or adapt it to the following sample:\n"
-fi
-
-# Sample commands for configuring and making SU2
-printf "A sample SU2 configure procedure might look like: ./configure --prefix=\$SU2_HOME --enable-mpi\n"
-printf "A sample SU2 make procedure might look like: make -j 4\n\n"
-
-printf "Adapter installation completed. Make sure to follow the instructions above to build the adapted SU2.\n\n"
```

Comment:
Since we are anyway using a specific version, let's document the instructions. I understand that they now are:

```bash
./meson.py build
./ninja -C build install
```

which automatically uses all cores.

Reply:
No particular reason for removing it except now that SU2 uses meson and ninja, and I just haven't bothered taking the time yet to figure out where the previously run meson settings had been saved. It may be super straightforward.

Yes usually it seems most of the time the above lines correctly set up SU2 (and w/ MPI if it's available).